ChangeLog.md

```diff
@@ -0,0 +1,44 @@
+# Changelog
+
+All notable changes to this project are documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
+
+## [Unreleased]
+
+## [2.00.2] - 2022-04-12
+### Added
+- added support for additional ArangoDB Collections API features (#356)
+- `GET/_api/collection/{collection-name}/checksum`: Returns checksum for the collection
+- `PUT/_api/collection/{collection-name}/loadIndexesIntoMemory`: Loads a collection's indexes into memory
+- `PUT/_api/collection/{collection-name}/recalculateCount`: Recalculate count of a collection
+- `PUT​/_api​/collection​/{collection-name}​/responsibleShard`: Returns the responsible shard for a document
+- `GET/_api/collection/{collection-name}/shards`: Returns the shard ids of a collection
+- `PUT/_api/collection/{collection-name}/compact`: Compacts a collection
+
+## [2.00.1] - 2022-03-24

```

Comment:
I'm confused by these version increments. All these changes haven't been release yet, why are they listed under a new version?

In addition, version numbers are usually written with a single digit when less than 10, e.g. `2.0.1`. Not a big deal but `2.00.1` looked off to me.

Reply:
Good points @DiscoPYF:
- We have removed version numbers since we have not yet released these features. We will add the new version 2.0.0 when we release.
- I agree with using single digits as well when the number is less than 10.